### DISCUSSION TITLE
为什么左侧栏菜单没有图标显示

### DISCUSSION THREAD

**Discussion by laotun-s** [solstice23/argon-theme#244] [Q&A]
![image](https://user-images.githubusercontent.com/55691621/127270722-a13501fc-cadb-4b7b-b112-5eade31c442d.png)

**Comment by solstice23** [comment]
https://argon-docs.solstice23.top/#/icon

**Reply by laotun-s** [reply]
好的，谢谢

**Reply by yimu56** [reply]
没有用啊<i class="fa fa-commenting-o"></i> 留言板
